source/slang/core.meta.slang

```diff
@@ -1664,6 +1664,18 @@ struct matrix : IRWArray<vector<T,C>>
     int getCount() { return R; }
 }
 
+__generic<T:__BuiltinArithmeticType, let N:int>
+__magic_type(CoopVectorExpressionType)
+__intrinsic_type($(kIROp_CoopVectorType))
+struct CoopVector : IArray<T>
```

Comment:
Should delete this too.

This PR contains: `struct CoopVector`, `struct CoopVec`, and `struct coopvecNV`. We just need `CoopVec`.